nvm.sh

```diff
@@ -2357,6 +2473,9 @@ nvm_die_on_prefix() {
   if [ -n "${NVM_NPM_CONFIG_PREFIX_ENV-}" ]; then
     local NVM_CONFIG_VALUE
     eval "NVM_CONFIG_VALUE=\"\$${NVM_NPM_CONFIG_PREFIX_ENV}\""
+    if [ -n "${NVM_CONFIG_VALUE-}" ] && [ "_${NVM_OS}" = "_win" ]; then
+      NVM_CONFIG_VALUE="$(nvm_ensure_posix_path "$NVM_CONFIG_VALUE")"
```

Comment:
I guess that in fact, as NVM_CONFIG_VALUE is probably supposed to be a folder, there might not even the need of nvm_ensure_posix_path, but it could be just done like that:
```shell
NVM_CONFIG_VALUE="$(cd "$NVM_CONFIG_VALUE" 2>/dev/null && pwd)"
```
This would be needed only for MSYS and Cygwin, not for WSL, so it would work.
Thanks to @pjeby for the  hint.

Reply:
sounds good to me.